content/blog/2016/2016-08-26-ask-the-experts-jenkins-world.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Ask the Experts at Jenkins World 2016"
+tags:
+- jenkinsworld
+- event
+author: rtyler
+---
+
+
+image:/images/conferences/Jenkins-World_125x125.png[role=right]
+
+Our events officer link:https://github.com/alyssat[Alyssa] has been working for
+the past number of weeks to organize the "Open Source Hub" at
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2016]. The Hub
+is a location on the expo floor where members of the Jenkins project can hang
```

Comment:
Am I a "member" of the project? Or a "contributor to" it? The latter seems more apt, but I don't know what is commonly used.